jmeter-service/chart/values.yaml

```diff
@@ -5,6 +5,7 @@ jmeterservice:
     tag: ""                                    # Container Tag
   service:
     enabled: true                              # Creates a Kubernetes Service for the jmeter-service
+  gracePeriod: 120                             # PreStop hook +30s
```

Comment:
same as with helm-service: why is the grace period configurable, but the prestop hook time isn't?